asset-transfer-basic/rest-api-typescript/src/config.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const logLevel = process.env.LOG_LEVEL || 'info';
```

Comment:
Suggestion - found the 'env-var' package very useful for adding control over the environment variables. 

_c.f._ https://github.com/hyperledgendary/contract-as-a-service/blob/06e61ed76d63a5edcb019bd2818782d41b6a307b/client-apps/metadata/index.js#L15

Reply:
Ah, that does look quite nice, thanks